lib/cli.ex

```diff
@@ -1,7 +1,20 @@
 defmodule Onigumo.CLI do
-  def main([component]) do
-    module = Module.safe_concat("Onigumo", component)
+  def main(argv) do
+    {_parsed, args, _invalid} = OptionParser.parse(argv, strict: [])
+
+    module =
+      args
+      |> get_component
+      |> safe_concat
+
     root_path = File.cwd!()
     module.main(root_path)
   end
+
+  defp get_component([]), do: "Downloader"
```

Comment:
This adds additional logic of setting a default component. That’s not necessarily wrong, at least now when there is only a single component, but it’s over the scope of introducing _OptionParser_.